uds/database/data_record/mapping_data_record.py

```
@@ -1,72 +1,103 @@
     @property
-    def mapping(self) -> MappingProxyType[int, str]:
-        """Get mapping dict."""
-        return self.__mapping
+    def values_mapping(self) -> MappingProxyType[int, str]:
+        """Get raw values mapping to their corresponding labels."""
+        return self.__values_mapping
 
-    @mapping.setter
-    def mapping(self, mapping: Dict[int, str]) -> None:
+    @values_mapping.setter
+    def values_mapping(self, value: Dict[int, str]) -> None:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)